src/swell/tasks/get_ensemble_geos_experiment.py

```diff
@@ -0,0 +1,131 @@
+# (C) Copyright 2021- United States Government as represented by the Administrator of the
+# National Aeronautics and Space Administration. All Rights Reserved.
+#
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+
+
+# --------------------------------------------------------------------------------------------------
+
+from datetime import datetime as dt
+import isodate
+import os
+import tarfile
+
+from swell.tasks.base.task_base import taskBase
+from swell.utilities.datetime_util import datetime_formats
+
+# --------------------------------------------------------------------------------------------------
+
+
+class GetEnsembleGeosExperiment(taskBase):
+
+    def execute(self):
+
+        """Acquires background files from a previous GEOS-FP experiment. This task
+        is specific to a 6-hour DA window and GEOS-FP experiment. Hence, it makes
+        certain assumptions about the file structure and naming conventions
+        (examples are provided for the x0048 experiment):
+
+            - Background archive file:       x0050.atmens_ebkgx.20231009_21z.tar
+            - First background file:         ensbkgx/mem001/x0050.bkg_clcv.20231009_2100z.nc4
+            .
+            .
+            .
+            - Last background file:          ensbkgx/mem001/x0050.bkg_clcv.20231010_0300z.nc4
+
+        The name of the *bkgcrst.*.tar files correspond to the forecast start time.
+        The file names in these tar archives corresponds to the background files
+        within the DA window, therefore 7 background files are extracted for a
+        6-hour window.
+
+        """
+
+        # Parse config
+        # ------------
+        background_experiment = self.config.background_experiment()
+        geos_x_ensemble_directory = self.config.geos_x_ensemble_directory()
+        # background_time_offset = self.config.background_time_offset()
+        background_time_offset = 'PT3H'
```

Comment:
This needs to be a properly configured default value.

Reply:
Thanks!  should have recovered it (I did it now), but it is also a reminder we need to merge the PR:
- https://github.com/GEOS-ESM/swell/pull/519

Reply:
Wait, I think it might be from @Dooruk, and it works well, but if it should be the default, I am not sure.

Reply:
```suggestion
        background_time_offset = self.config.background_time_offset()
```

should work, as long as `task_questions.yaml` is set for `GetEnsembleGeosExperiment` to access `background_time_offset`.

Reply:
@Dooruk: It works when overriding background_time_offset in localensembleda-tier1.yaml. The default value is PT9H and the setup varies in 3dvar-tier1.yam vs 3dfgat_cycle-tier1.yaml vs 3dvar_cycle-tier1.yaml.